### PR TITLE
Add search keywords to reviewer tools (bug 1139525)

### PR DESCRIPTION
**Pull request by robhudson**

### CODE REVIEW

File: mkt/reviewers/templates/reviewers/includes/mobile_details.html [hunk 1]
```diff
@@ -136,6 +138,8 @@
             <dt>{{ _('Flags') }}</dt>
             <dd>{{ flags|join(', ') }}</dd>
           {% endif %}
+          <dt>{{ _('Region Availability') }}</dt>
+          <dd>{{ product.get_regions(sort_by='name')|map(attribute='name')|join(', ') }}</dd>
```
Comment (andymckay) [inline mkt/reviewers/templates/reviewers/includes/mobile_details.html]: looks like this is a different bug?

**Comment by andymckay** [comment]
no tests needed? but r+

File: mkt/reviewers/templates/reviewers/includes/mobile_details.html (hunk 1 above)
Comment (robhudson) [inline mkt/reviewers/templates/reviewers/includes/mobile_details.html]: This was the mobile template not up to date with the regular template.

**Comment by robhudson** [comment]
https://github.com/mozilla/zamboni/commit/db6976a

**Pull request closed without merging**